.gitignore

```diff
@@ -3,3 +3,5 @@
 /telegraf.exe
 /telegraf.gz
 /vendor
+.idea/
+
```

Comment:
Revert this file, you can put editor specific ignores in your git excludesfile, something like this:
```
$ git config global core.excludesfile = "/home/smicha/.gitignore"
$ echo ".idea/" >> /home/smicha/.gitignore